src/xdocs/config_filters.xml

```diff
@@ -632,6 +633,160 @@ public class UserService {
         <p> <a href="config.html#Checker">Checker</a> </p>
       </subsection>
     </section>
+    <section name="SuppressionXpathFilter">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.2</p>
+        <p>
+            Filter <code>SuppressionXpathFilter</code> rejects
+            TreeWalker audit events for Check errors according to
+            a <a href="config.html#XML_Structure"><em>suppressions XML
+            document</em></a> in a file. If there is no configured
+            suppressions file or the optional is set to true and
+            suppressions file was not found the Filter accepts all audit events.
+            Filter processes only xpath suppressions from suppressions file, with tag
+            <code>&lt;suppress-xpath&gt;</code>.
+        </p>
+      </subsection>
+      <subsection name="Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>file</td>
+            <td>
+              the location of the <em>suppressions XML document</em> file.
+              The order the location is checked is:
+              <ol>
+                <li>as a filesystem location</li>
+                <li>
+                  if no file found, and the location starts with either
+                  <code>http://</code> or <code>https://</code>, then it
+                  is interpreted as a URL
+                </li>
+                <li>
+                  if no file found, then passed to the
+                  <code>ClassLoader.getResource()</code> method.
+                </li>
+              </ol>
+            </td>
+            <td><a href="property_types.html#string">string</a></td>
+            <td><code>none</code></td>
+            <td>3.2</td>
```

Comment:
version is not correct. It should be equal to https://github.com/checkstyle/checkstyle/pull/4992/files#diff-7d423a3c3d58607e2748c950c04d32d4R638. Same is for other properties.

Reply:
done